### ISSUE TITLE
Modo de uso

### ISSUE THREAD

**Issue by GerardoMunoz** [GerardoMunoz/roi-videos#3]
Yo propongo que cuando se oprima el botón izquierdo se detenga el video y permita dibujar un rectángulo mientras está oprimido el botón. Cuando se levanta el botón izquierdo entonces guarda un archivo con la imagen que hay dentro del rectángulo. 

Si está de acuerdo con ese modo de uso,  por favor escriba como se puede implementar en un nuevo aporte (*issue*). 

Si no está de acuerdo o tienen un aporte, por favor escríbalo en este aporte (*issue*).

**Comment by Chaux14** [comment]
Me parece bien la idea, lo que yo cambiaria seria que se empleara una letra (p) como pause y al presionar otra (c) se habilite el rectángulo de captura.

**Comment by voccuer** [comment]
Me parece buena idea, yo le agregaría una opción para reajustar el tamaño del rectángulo (en caso de que no haya quedado como uno quisiera), y confirmar si se quiere guardar.

**Comment by AlejaDT** [comment]
Estoy de acuerdo con la idea, pero me gustaría que se cambiara la forma de detección de imagen a circular, para ser más precisos. Por otra parte, también noté que al reproducir los videos, estos van muy rápido, entonces sería pertinente configurar los frames por segundo. Además sería interesante que la interfaz de video tuviese una serie de botones que permita realizar la captura de la imagen, guardar la imagen, poner o quitarle el color a los videos, aumentar su velocidad y configurar su sonido.

**Comment by JulianCarrero** [comment]
Me parece que la idea es buena, sin embargo, antes de guardar cualquier imagen me parece mejor crear un pequeño cuadro de dialogo que permita modificar el tamaño de asignación en la captura de la pantalla en conjunto con opciones como: copiar, guardar y descartar la imágen.

**Comment by Dadadaadad** [comment]
Me parecen muy buenas ideas, yo propondria que despues de hacer la seleccion, poder mover por fotogramas el video con las letras (K) para retroceder y (L) para avanzar. Para que de esta forma se pueda buscar una imagen mas exacta, la cual se guardaria en la base de datos
